.evergreen/config.yml

```diff
@@ -172,101 +172,122 @@ functions:
 
 pre:
   - func: "fetch source"
-  - func: "prepare resources"
```

Comment:
`prepare resources` downloads drivers evergreen tools and starts mongo orchestration, so we don't want to do it before all tasks anymore as its irrelevant for atlas tests.